readme.md

```diff
@@ -135,6 +155,7 @@ The debug will show you an array like this:
 
 ### Differences
 
+* No authentication layer is provided, you will need to build this yourself.
```

Comment:
IMO there is no need to mention that. It's obvious and was mentioned before as well.